packages/@coorpacademy-translate/src/translate.js

```diff
@@ -16,21 +16,25 @@ function getTemplate(locales, key, count) {
   return getOr(regularTemplate, `${key}_plural`, locales);
 }
 
-const createTranslate = locales => (key, data) => {
-  const template = getTemplate(locales, key, get('count', data));
-  if (!isString(template)) {
-    throw new Error(`Key ${key} not found!`);
-  }
+const createTranslate =
+  (locales, isAllowFalsy = false) =>
```

Comment:
I'd give a more functional name to the arg, like `throwIfMissing` (which invert the values)
```suggestion
  (locales, throwIfMissing = true) =>
```